### ISSUE TITLE
could you open source the model file

### ISSUE THREAD

**Issue by SifengHe** [xwen99/temporal_context_aggregation#1]
Hi, thanks for your work! We have tried to train the model based on your training code. However, we could not achieve the same metric as your paper. Could you open source your model file or give us more instructions? Thanks a lot!

**Comment by xwen99** [comment]
Hi, thank you for your concern. However, I have left ByteDance for a while, so the model may be hard to find.
To achieve better metric, please check if all the hyper-parameters are aligned to the paper, not only in the training stage, and also in the pre-processing stage (for example, the default parameters use imac feature, and the final results in the paper are based on rmac feature).
If you can share more details about your re-implementation, I might be able to offer more help.
